data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -628,17 +632,40 @@ def _workforce_factor(self) -> bool:
             >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
         )
 
-        workforce_combined_criteria_for_island_areas = (
-            self.df[unemployment_island_areas_criteria_field_name]
-            | self.df[poverty_island_areas_criteria_field_name]
-            | self.df[
-                low_median_income_as_a_percent_of_ami_island_areas_criteria_field_name
-            ]
-        ) & (
+        island_areas_high_scool_achievement_rate_threshold = (
             self.df[field_names.CENSUS_DECENNIAL_HIGH_SCHOOL_ED_FIELD_2009]
             >= self.LACK_OF_HIGH_SCHOOL_MINIMUM_THRESHOLD
         )
 
+        self.df[
+            field_names.ISLAND_AREAS_UNEMPLOYMENT_LOW_HS_EDUCATION_FIELD
+        ] = (
+            self.df[island_areas_unemployment_criteria_field_name]
+            & island_areas_high_scool_achievement_rate_threshold
+        )
+
+        self.df[field_names.ISLAND_AREAS_POVERTY_LOW_HS_EDUCATION_FIELD] = (
+            self.df[island_areas_poverty_criteria_field_name]
+            & island_areas_high_scool_achievement_rate_threshold
+        )
+
+        self.df[
+            field_names.ISLAND_AREAS_LOW_MEDIAN_INCOME_LOW_HS_EDUCATION_FIELD
+        ] = (
+            self.df[
+                island_areas_low_median_income_as_a_percent_of_ami_criteria_field_name
+            ]
+            & island_areas_high_scool_achievement_rate_threshold
+        )
+
+        workforce_combined_criteria_for_island_areas = self.df[
+            island_areas_workforce_eligibility_columns
+        ].any(axis="columns")
+
+        self._increment_total_eligibility_exceeded(
```

Comment:
Actually increment the "criteria exceeded" column for island areas (this was not done before).

Reply:
Okay - this makes sense. This is also related to the issue #983. We had discussed handling the islands separately from the states.